src/Transloadit.ts

```diff
@@ -723,15 +781,21 @@ class TransloaditClient {
 
         const shouldRetry =
           statusCode === 413 &&
+          typeof body === 'object' &&
+          body != null &&
+          'error' in body &&
           body.error === 'RATE_LIMIT_REACHED' &&
-          body.info &&
-          body.info.retryIn &&
+          'info' in body &&
+          typeof body.info === 'object' &&
+          body.info != null &&
+          'retryIn' in body.info &&
+          Boolean(body.info.retryIn) &&
```

Comment:
maybe if we add this, then TS can properly infer that body.info.retryIn is a number, and we can remove the ` as { retryIn: number }` below? (which is kind of a lie now)
```suggestion
          typeof body.info.retryIn === 'number' &&
          Boolean(body.info.retryIn) &&
```

Reply:
That didn’t work, but I wasn’t a fan of the construct myself either. I rewrote it to smaller checks that throw early.

Reply:
That didn’t work as I intended, so I reverted it.

Reply:
I took the liberty of fixing this